lock-api-objects/src/main/java/com/palantir/lock/v2/AcquireNamedMinTimestampLeaseResult.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.lock.v2;
+
+import com.palantir.timestamp.TimestampRange;
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface AcquireNamedMinTimestampLeaseResult {
```

Comment:
I'm removing `minimum` in favor of `min` everywhere per your recommendation

Reply:
Would this eventualy be a conjure object?

Reply:
Not in this case. Or, at least, this won't be the same as the conjure object in the timelock server side of things. For one, this package does not depend on it (and, I think it's better to keep it that way). It defines most of its API objects here

Also, they might not match due to https://github.com/palantir/atlasdb/pull/7327#discussion_r1791617199 if it makes sense to you